1-JavaScript/Week3/assignment/ex2-mondaysWorth.test.js

```diff
@@ -31,10 +31,16 @@ const mondayTasks = [
 
 const hourlyRate = 25;
 
-function computeEarnings(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function computeEarnings(tasks, rate) {
+  const totalEarnings=  tasks
```

Comment:
Small spacing issue here. 

```suggestion
  const totalEarnings = tasks
```